shopfloor/services/zone_picking.py

```diff
@@ -316,6 +349,16 @@ def _group_buffer_move_lines_by_package(self, move_lines):
             data[move_line.result_package_id] |= move_line
         return data
 
+    def select_zone(self):
```

Comment:
Question: what is the state we are coming from when calling `select_zone`?
Before we were starting from `start` and we call `scan_location`, but now `start` isn't anymore the first state if I'm not wrong?

Reply:
for the client "start" is an alias for the 1st state. In this case "scan_location" is the 1st state.
I prefer to keep "select_zone" because it has more meaning and is not the 1st state anyway, is an helper for the 1st state.